docs/proposals/scheduling/20221227-cpu-static-reservation.md

```diff
@@ -0,0 +1,176 @@
+---
+title: resource reservation
```

Comment:
> > remember to change the title (might be `CPU Static Reservation`)
> 
> @saintube We expect to reserve not only CPU, but also other resources.

Does the `resource reservation` duplicate the current implemented `Resource Reservation`? Maybe `Static Resource Reservation` is better.

Reply:
Maybe `Node Static Resource Reservation` is better, because these are node level.
refer https://github.com/kubernetes/design-proposals-archive/blob/main/node/node-allocatable.md